src/hotspot/share/oops/stackChunkOop.inline.hpp

```diff
@@ -213,15 +252,14 @@ inline void stackChunkOopDesc::iterate_stack(StackChunkFrameClosureType* closure
                          RegisterMap::ProcessFrames::skip,
                          RegisterMap::WalkContinuation::include);
     full_map.set_include_argument_oops(false);
+    closure->do_frame(f, map);
```

Comment:
This could use a comment.  I guess we weren't looking at the stub frame before, only the caller.  Why is this using `map` instead of `full_map`?